.github/workflows/php.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         operating-system: [ubuntu-latest]
-        php-versions: ['7.3', '7.4', '8.0']
+        php-versions: ['7.3', '7.4', '8.0', '8.1', '8.2', '8.3']
```

Comment:
I think only the last version of php7 should be left - php7.4. 
And all version of php8